scrypto/src/component/stubs.rs

```diff
@@ -150,18 +151,18 @@ extern_blueprint_internal! {
             owner_role: OwnerRole,
             address_reservation: Option<GlobalAddressReservation>,
         ) -> Global<Account>;
-        fn create() -> (Global<Account>, Bucket);
+        fn create() -> (Global<Account>, NonFungibleBucket);
     },
     {
-        fn securify(&mut self) -> Bucket;
+        fn securify(&mut self) -> NonFungibleBucket;
         fn lock_fee(&mut self, amount: Decimal);
         fn lock_contingent_fee(&mut self, amount: Decimal);
         fn deposit(&mut self, bucket: Bucket);
         fn deposit_batch(&mut self, buckets: Vec<Bucket>);
         fn withdraw(&mut self, resource_address: ResourceAddress, amount: Decimal) -> Bucket;
         fn withdraw_non_fungibles(
             &mut self,
-            resource_address: ResourceAddress,
+            resource_address: NonFungibleBucket,
```

Comment:
Probably an accidental mistake, but this should remain to be a `ResourceAddress`.

Reply:
It looks like this is an issue with how the function signatures were overriden in `update-bindgen.sh`.